### ISSUE TITLE
Request and read packet timestamps

### ISSUE THREAD

**Issue by Georacer** [ZdenekM/microstrain_3dm_gx3_45#9]
If I read the code correctly, you stamp the ROS topics with the current system time, using the C++ clock_gettime.

Is there a reason you use the local system time instead of the packet timestamp (when that is available)?

**Comment by ZdenekM** [comment]
You are right. And no - there is no special reason for that and it is not correct. Feel free to open a pull request improving this.

**Comment by Georacer** [comment]
Okay, just wanted to have a sanity check. Thanks!
